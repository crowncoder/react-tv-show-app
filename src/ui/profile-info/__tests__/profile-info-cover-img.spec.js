import React from 'react';
import Enzyme, { shallow } from 'enzyme'
import EnzymeAdapter from 'enzyme-adapter-react-16'
import ProfileInfoCoverImg from '../profile-info-cover-img'

Enzyme.configure({ adapter: new EnzymeAdapter() });

describe('<ProfileInfoCoverImg /> unit test', () => {
  it('renders correctly', () => {
    const imageInfo = { medium: 'abc.com/image' };
    const wrapper = shallow(<ProfileInfoCoverImg image={imageInfo} />);
    expect(wrapper.find('img').prop('src')).toEqual('abc.com/image');
    expect(wrapper).toMatchSnapshot();
  });
});