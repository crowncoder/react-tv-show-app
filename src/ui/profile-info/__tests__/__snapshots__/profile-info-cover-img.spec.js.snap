// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileInfoCoverImg /> unit test renders correctly 1`] = `
<figure
  className="profileInfo__figure"
>
  <img
    alt="The Powerpuff Girls"
    src="abc.com/image"
  />
</figure>
`;
